rules.mk

```diff
@@ -157,26 +157,27 @@ go.tidy:
 	  $(GO)	mod tidy; \
 	); done
 
-.PHONY: go.depaware-update
-go.depaware-update: go.tidy
-	@# gen depaware for bins
-	@set -e; for dir in $(GOBINS); do ( set -xe; \
-	  cd $$dir; \
-	  $(GO) run github.com/tailscale/depaware --update .; \
-	); done
-	@# tidy unused depaware deps if not in a tools_test.go file
-	@set -e; for dir in $(GOMOD_DIRS); do ( set -xe; \
-	  cd $$dir; \
-	  $(GO)	mod tidy; \
-	); done
 
-.PHONY: go.depaware-check
-go.depaware-check: go.tidy
-	@# gen depaware for bins
-	@set -e; for dir in $(GOBINS); do ( set -xe; \
-	  cd $$dir; \
-	  $(GO) run github.com/tailscale/depaware --check .; \
-	); done
```

Comment:
please revert this part of the code

Reply:
Yes sorry about that, we disabled it because we couldn't fix the following error:
[![Image from Gyazo](https://i.gyazo.com/bd6ab1057c71d2acf38259b560ea15d9.png)](https://gyazo.com/bd6ab1057c71d2acf38259b560ea15d9)